static/js/Router.js

```diff
@@ -1,29 +0,0 @@
-import React from 'react';
```

Comment:
No, we decided to not make this a single page app so routing will be done by Django directly